scripts/dts/extract/flash.py

```diff
@@ -99,7 +129,12 @@ def extract_partition(self, node_path):
         prop_alias = {}
         node = reduced[node_path]
 
+<<<<<<< HEAD
         self._extract_partition(node_path)
+=======
+        self._extract_partition(node_address)
+        self._extract_partition_dev(node_address)
+>>>>>>> scritps/dts: Build a list of flash area device names
```

Comment:
Accidentally committed merge conflict. :)

Reply:
@ulfalizer, thanks a lot for the review. I have to say, I don't really understand how these scripts are working. I just copied the _extract_partition and removed everything I didn't need.

Reply:
@Laczen 
Yeah, no worries. Many people who haven't done that much Python have been working on these, so there's some weirdness in there. :)

Reply:
I'm trying to make them a bit easier to read. Many parts are super hard for me to understand too.